logstash-core/lib/logstash/pipeline.rb

```diff
@@ -21,24 +21,24 @@
 
 module LogStash; class Pipeline
   attr_reader :inputs,
-    :filters,
-    :outputs,
-    :worker_threads,
-    :events_consumed,
-    :events_filtered,
-    :reporter,
-    :pipeline_id,
-    :logger,
-    :started_at,
-    :thread,
-    :config_str,
-    :settings,
-    :metric
+      :filters,
+      :outputs,
+      :worker_threads,
+      :events_consumed,
+      :events_filtered,
+      :reporter,
+      :pipeline_id,
+      :logger,
+      :started_at,
+      :thread,
+      :config_str,
+      :settings,
+      :metric
 
   MAX_INFLIGHT_WARN_THRESHOLD = 10_000
 
   RELOAD_INCOMPATIBLE_PLUGINS = [
-    "LogStash::Inputs::Stdin"
+      "LogStash::Inputs::Stdin"
```

Comment:
IntelliJ Ruby support has continuation indent of 4 spaces. I think this is acceptable.